examples/getstarted/package.json

```diff
@@ -13,7 +13,7 @@
     "strapi": "strapi"
   },
   "dependencies": {
-    "@strapi/icons": "2.0.0-rc.21",
+    "@strapi/icons": "2.0.0-rc.22",
```

Comment:
The DS bump should be a standalone PR

Reply:
It will be, I'll pull to update

Reply:
yes all the dependencies are now removed